### DISCUSSION TITLE
Creating the template

### DISCUSSION THREAD

**Discussion by digaomatias** [iayti/CleanArchitecture#25] [General]
Hey mate, I noticed you created a nuget template for this project.
Could you give me some directions how could I make a template nuget package out of my fork?

Thanks mate.

**Comment by iayti** [comment]
> Hey mate, I noticed you created a nuget template for this project.
> Could you give me some directions how could I make a template nuget package out of my fork?
> 
> Thanks mate.

Hi @digaomatias 

I have a repository for [sample nuget template.](https://github.com/iayti/Matech.Sample.Template)

You can find the information you need in the README. I'll try to do my best if you need extra help.

Have a nice coding 😊
